pkg/generator/dspace_test.go

```diff
@@ -44,6 +44,14 @@ func TestDspaceRecordParsing(t *testing.T) {
 		t.Error("Expected 'Price, Max et al. \"Fodder, pasture, and the development of complex society in the Chalcolithic: isotopic perspectives on animal husbandry.\" Archaeological and Anthropological Sciences 12, 4 (March 2020): 95 © 2020 Springer-Verlag', got", record.Citation)
 	}
 
+	// Test Collection field
+	if len(record.Collection) != 1 {
+		t.Error("Expected 1, got", record.Collection)
+	}
+	if record.Collection[0] != "MIT Open Access Articles" {
```

Comment:
We can. We're not doing that degree of testing for any other field for any of our source parsers. Rather than just add it here randomly for Collections from DSpace, I'd like to create a new ticket for us to expand our full test suite to ensure that we're handling likely expected data anomalies from each of our sources, assuming we want to do that. Thoughts?

Reply:
Added a ticket for this.